docs/index.rst

```diff
@@ -18,36 +18,57 @@ ops.charm module
 ----------------
 
 .. automodule:: ops.charm
+   :members:
```

Comment:
seems unfortunate to have to repeat yourself for each one vs inheriting from a common base.

Reply:
I agree. I'm going to see if I can back out the change -- I suspect the actual error was masking the problem.